beacon_node/beacon_chain/src/custody_context.rs

```diff
@@ -623,16 +593,17 @@ mod tests {
             ssz_context,
             NodeCustodyType::Fullnode,
             head_epoch,
+            generate_custody_column_indices(),
             spec,
         );
 
-        let all_custody_groups_ordered = (0..spec.number_of_custody_groups).collect::<Vec<_>>();
-        custody_context
-            .init_ordered_data_columns_from_custody_groups(all_custody_groups_ordered, spec)
-            .expect("should initialise ordered data columns");
         custody_context
     }
 
+    fn generate_custody_column_indices() -> Vec<u64> {
+        (0..E::number_of_columns() as u64).collect::<Vec<_>>()
```

Comment:
All these tests use `0..number_of_columns`, which kind of implies the "ordering" is ordering by ascending column. But as I understand it (looking at `get_custody_groups_ordered`) the ordering is actually the ordering the columns are produced in from the `node_id`. Maybe we should use a "more arbitrary" ordering here, to make sure we're not accidentally depending on the ascending ordering?

Reply:
Looks good!